components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfig.java

```diff
@@ -77,8 +77,6 @@ public boolean isExecutionForActionTypeEnabled(ActionType actionType) {
         switch (actionType) {
             case PRE_ISSUE_ACCESS_TOKEN:
                 return isActionTypeEnabled(ActionTypeConfig.PRE_ISSUE_ACCESS_TOKEN.getActionTypeEnableProperty());
-            case AUTHENTICATION:
```

Comment:
Why we are removing this. is this for enabling/diasabling authenticatiomn action

Reply:
added